integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/IstioUtils.java

```diff
@@ -363,6 +395,27 @@ public static Domain createIstioDomainResource(String domainUid, String domNames
                     .onlineUpdate(new OnlineUpdate().enabled(true))
                     .runtimeEncryptionSecret(encryptionSecretName))
                 .introspectorJobActiveDeadlineSeconds(300L)));
+    if (monexpConfig != null) {
+      LoggingFacade logger = getLogger();
+      logger.info("yaml config file path : " + monexpConfig);
+      String contents = null;
+      try {
+        contents = new String(Files.readAllBytes(Paths.get(monexpConfig)));
+      } catch (IOException e) {
+        e.printStackTrace();
```

Comment:
If there is a exception in reading the config file don't you need to fail?

Reply:
fixed